### PR TITLE
Replace email with Slack in pxf-build

### PR DESCRIPTION
**Pull request by bradfordb-vmware**

### CODE REVIEW

**Review by ashuka24 — APPROVED** [review]

File: concourse/Makefile [hunk 1]
```diff
@@ -125,7 +122,6 @@ set-dev-build-pipeline:
 		--config "$${PIPELINE_FILE}" \
 		--load-vars-from=$(HOME)/workspace/pxf/concourse/settings/pxf-multinode-params.yml \
 		--var=pxf-git-branch=${BRANCH} \
-		--var=slack-integration-webhook-url=$${SLACK_INTEGRATION_WEBHOOK_URL} \
```
Comment (denalex) [inline concourse/Makefile:128]: why was this removed ?

Comment (bradfordb-vmware) [reply concourse/Makefile:128]: I removed this because the variable (`slack-integration-webhook-url`) isn't actually used. Looking at the git history, this variable wasn't actually ever used.

**Review by yimingli-vmware — APPROVED** [review]

**Review by denalex — APPROVED** [review]
